ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/_cq_dialog/.content.xml

```diff
@@ -120,6 +120,15 @@
                                               jcr:primaryType="nt:unstructured"
                                               text="Form Data Model"
                                               value="formdatamodel"/>
+                                            <optionMarketomodel
+                                              jcr:primaryType="nt:unstructured"
+                                              text="Marketo Configuration"
+                                              value="connector">
+                                              <granite:rendercondition
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/renderconditions/featuretoggle"
+                                                toggleName="FT_FORMS-9611"/>
```

Comment:
FT cannot be part of core component, can you please create a render condition in cq-guides and re-use the resource type here.

ideally, core component should not contain logic which you have written specific to marketo, this should be part of cq-guides

Reply:
done, render condition has been moved in cq-guides and using the resource type here.
Not specific to marketo any logic is here, its new type of form model which is in-memory direct connector